redis.go

```diff
@@ -24,15 +25,42 @@ type Redis struct {
 	Ttl            uint32
 	Zones          []string
 	LastZoneUpdate time.Time
+	lastKeyCount   int
 }
 
+func (redis *Redis) KeyCount() int {
+	var (
+		reply interface{}
+		err error
+	)
+	conn := redis.Pool.Get()
+	if conn == nil {
+		fmt.Println("error connecting to redis")
+		return -1;
+	}
+	defer conn.Close()
+	reply, err = conn.Do("DBSIZE")
+	if err != nil {
+		fmt.Println("error getting dbsize from redis:", err)
+		return -1;
+	}
+	dbsize, err := redisCon.Int(reply, nil)
+	if err != nil {
+		fmt.Println("error parsing dbsize:", err)
+		 return -1
+	}
+	return dbsize;
+} 
+
 func (redis *Redis) LoadZones() {
 	var (
 		reply interface{}
 		err error
 		zones []string
 	)
 
+	fmt.Println("loading zones")
```

Comment:
This ends up logging every two minutes, and could just be a `log.Debug` instead.